admin/schema.php

```diff
@@ -50,11 +50,7 @@ function db_null_date() {
 function installer_db_now() {
 	global $g_db;
 
-	$t_timezone = @date_default_timezone_get();
-	date_default_timezone_set( 'UTC' );
-	$t_time = $g_db->BindTimeStamp( time() );
-	@date_default_timezone_set( $t_timezone );
-	return $t_time;
+	return $g_db->BindTimeStamp( time() );
```

Comment:
Let's add a code comment in this line to avoid making the same mistake again in the future.